docker/compose/docker-compose.yml

```diff
@@ -66,6 +66,7 @@ services:
     command: controller
     environment:
       WAIT_FOR: zookeeper:2181
+      ZK_URL: zookeeper:2181
```

Comment:
This looks ok, but I'm wondering about the `zookeeper` host name. Is it correct?

Reply:
Yes, @fpj, the `zookeeper` host name is configured a few lines below in the `links` block, making the `zookeeper` host name available to the Controller.

Reply:
To add a bit more color, `zookeeper` is the name of the service defined later in the docker-compose.yml file. That service is linked to the Controller service via the `controller` service's `links` option. Since containers for linked services are reachable at a hostname identical to the alias or the service name, the hostname for zookeeper is indeed `zookeeper`.

Reply:
To verify the discussion about the hostname, I ran a few commands to check the `hostname` of the Zookeeper container on a deployment. (The name of the container was `compose_zookeeper_1`)

```
 (Run the hostname command inside the Zookeeper container)
..:~/..#  docker exec -it compose_zookeeper_1 hostname
99665322b64f
```
The above shows that the hostname visible inside the container is `99665322b64f`. 

Then, I ran the following commands: 
```
(Start a shell inside another container, so that we are in a container which shares the same  
network as the Zookeeper container.)
# docker exec -it compose_controller_1 sh

(Now, ping `zookeeper`. If the host exists, we should see a typical ping response. ) 
# ping zookeeper
PING zookeeper (172.20.0.2): 56 data bytes
64 bytes from 172.20.0.2: seq=0 ttl=64 time=0.067 ms
64 bytes from 172.20.0.2: seq=1 ttl=64 time=0.204 ms
```
So, our assertions are correct.